docs/api.rst

```diff
@@ -962,6 +962,53 @@ Adding information for web3 nodes
    :statuscode 409: No user is logged or failed to delete because the node name is not in the database.
    :statuscode 500: Internal rotki error
 
+.. http:post:: /api/(version)/blockchains/(blockchain)/nodes
+
+   By doing a POST on this endpoint the app will try to connect to the provided node. If no node is provided then the app will try to connect to all the nodes tracked for the chain.
```

Comment:
Would it not make sense, as discussed in discord to turn this into a connect/disconnect endpoint?

And to make it so that the endpoint says, you are aleady connected if connected, or disconnected if not connected.